Compile warning on Perl 5.30

https://travis-ci.org/Dual-Life/Devel-PPPort/jobs/544798963
```
cc -c   -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2   -DVERSION=\"3.52\" -DXS_VERSION=\"3.52\" -fPIC "-I/home/travis/perl5/perlbrew/perls/5.30.0/lib/5.30.0/x86_64-linux/CORE"  -W -Wall -Wdeclaration-after-statement RealPPPort.c
In file included from RealPPPort.xs:142:0:
ppport.h: In function ‘DPPP_my_utf8_to_uvchr_buf’:
ppport.h:4873:12: warning: unused variable ‘overflow_length’ [-Wunused-variable]
     STRLEN overflow_length = 0;
            ^
```
ping @khwilliamson

--- a/parts/inc/uv
+++ b/parts/inc/uv
@@ -133,7 +133,9 @@ utf8_to_uvchr_buf(pTHX_ const U8 *s, const U8 *send, STRLEN *retlen)
     bool overflows = 0;
     const U8 *cur_s = s;
     const bool do_warnings = ckWARN_d(WARN_UTF8);
+#    if { VERSION < 5.26.0 } && ! defined(EBCDIC)
     STRLEN overflow_length = 0;
+#    endif
 
     if (send > s) {
         curlen = send - s;

